.github/workflows/database-athena-integration-test.yml

```diff
@@ -72,4 +72,4 @@ jobs:
           MAVEN_OPTS: -Xmx4g
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_INTEGRATION_USER1_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_INTEGRATION_USER1_SECRET_ACCESS_KEY }}
-        run: mvn -pl legend-engine-xt-relationalStore-athena-execution-tests -Dtest=ExternalIntegration*Athena test
+        run: mvn -pl legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-athena/legend-engine-xt-relationalStore-athena-execution-tests -Dtest=ExternalIntegration*Athena test
```

Comment:
Would be less verbose to use module names like so `mvn -pl :legend-engine-xt-relationalStore-athena-execution-tests` instead of having to use directory names